### ISSUE TITLE
Python 3.8 warnings

### ISSUE THREAD

**Issue by casatir** [pyodide/pyodide#877]
[Python issue #34850](https://bugs.python.org/issue34850) introduces a warning in Python 3.8 about missusing the `is` (or `is not`) statement.

In Pyodide, for some several 'old' modules we can get this warning. This occurs at least for `numpy` and `matplotlib`. This shows up in `console.html`with #875

```python
import numpy

/lib/python3.8/site-packages/numpy/ma/core.py:4467: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if self.shape is ():

import matplotlib.pyplot

/lib/python3.8/site-packages/matplotlib/pyplot.py:2422: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if fignum is False or fignum is 0:
/lib/python3.8/site-packages/matplotlib/contour.py:870: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  if self.extend is not 'neither':
```

There may be other warnings introduced in 3.8. One should identify theme and start upgrading impacted modules though.

**Edit:** list of packages that needs updating
 - [x] numpy
 - [x] matplotlib
 - [x] mpmath
 - [ ] ...

**Comment by rth** [comment]
yes, it will certainly become a major issue once we are able to see warnings in `console.html` :) 

We should be able update packages (except for scipy cf https://github.com/iodide-project/pyodide/issues/549) to a version post Oct. 14, 2019 (Python 3.8 release date).   Typically they should have a Python 3.8 compatible release by then. Unless some are blocked by out outdated scipy version (e.g. scikit-learn, scikit-image, statsmodels). This mostly concerns compiled packages with patches, all the rest should be already up to date.

**Comment by rth** [comment]
PR for numpy done in https://github.com/iodide-project/pyodide/pull/887

**Comment by rth** [comment]
There is also,
```
/lib/python3.8/site-packages/pandas/compat/__init__.py:117: UserWarning: Could not import the lzma module. Your installed Python is incompl
ete. Attempting to use lzma compression will result in a RuntimeError.
```
for `pandas` which is a bit strange, since lzma is a non mandatory build option.

**Comment by rth** [comment]
I believe all of these were fixed in Pyodide 0.20.0.

The pandas warning was also fixed in pandas 1.4.0+ (which we ship) since https://github.com/pandas-dev/pandas/pull/43495
